osi_logicallane.proto

```diff
@@ -0,0 +1,502 @@
+    //   marking appears or disappears.
+    //
+    optional Identifier physical_lane_id = 4;
+
+    // The reference line for this logical lane
+    //
+    // The reference line is used as a coordinate system on this lane.
+    //
+    // The reference line should roughly have the same shape as the lane, so
+    // that S coordinates continually increase/decrease along the lane. It is
+    // not required that the reference line has the same direction as the
+    // driving direction of the lane.
+    //
+    // Neighbouring lanes are strongly encouraged to use the same
+    // reference_line, so that vehicles that are next to each other on
+    // neighbouring lanes have comparable S positions.
+    //
+    optional Identifier reference_line_id = 5;
+
+    // Start S position of the lane. Must be in range [sStart,sEnd] of the
+    // reference line.
+    //
+    optional double start_s = 6;
+
+    // End S position of the lane. Must be in range [sStart,sEnd] of the
+    // reference line.
+    // end_s > start_s
+    //
+    optional double end_s = 7;
+
+    // Definition of the intended driving direction.
+    //
+    // \c true means driving direction is according to ascending S position.
+    // \c false means driving direction is according to descending S position.
+    //
+    optional bool reference_line_is_driving_direction = 8;
+
+    // Lanes that are directly right of this lane, without gap or overlap.
+    // "Right" is in definition direction (not driving direction), so right lanes
+    // have smaller T coordinates.
+    // Entries must be ordered: first by start_s, then by end_s.
+    //
+    repeated LaneRelation right_adjacent_lane = 9;
+
+    // Lanes that are directly left of this lane, without gap or overlap.
+    // "Left" is in definition direction (not driving direction), so left lanes
+    // have larger T coordinates.
+    // Entries must be ordered: first by start_s, then by end_s.
+    //
+    repeated LaneRelation left_adjacent_lane = 10;
+
+    // Lanes that partially or completely overlap this lane. Only overlaps
+    // laterally larger than 5cm are considered overlaps for the purpose of
+    // this relation.
+    // Entries must be ordered: first by start_s, then by end_s.
+    //
+    repeated LaneRelation overlapping_lane = 11;
+
+    // Right boundary of this lane.
+    // References to LogicalLaneBoundary elements.
+    // All LogicalLaneBoundary elements referenced here must refer to the same
+    // ReferenceLine as this lane.
+    // The boundaries together must cover the whole length of the lane (the
+    // range [start_s,end_s]) without gap or overlap. The boundaries must be
+    // stored in ascending order, starting with the smallest S position.
+    // Consecutive boundaries must share a point: the last point of the
+    // previous boundary must be identical to the first point of the next
+    // boundary.
+    //
+    // TODO Is it allowed for the boundaries to be longer than the lane?
```

Comment:
We would allow to have boundaries longer than lanes. Esther will make an example.

Reply:
Makes sense. Adapted accordingly.